visualizer-server/build.rs

```diff
@@ -16,6 +16,8 @@ fn compile(
     config
         .service_generator(generator)
         .compile_well_known_types()
+        // Rust cosider some comments as doc-comments and include them in tests
+        .disable_comments(["."])
```

Comment:
may be we could ignore generated files in doc tests instead?

Reply:
ok, added `doctest = false` to `cargo.toml`